generated/nidcpower/ctypes_types.py

```diff
@@ -0,0 +1,69 @@
+import ctypes
```

Comment:
This file was deleted in a different PR, so you shouldn't be adding it back.

Reply:
Pulled latest and updated generated